tests/net/lib/coap/src/main.c

```diff
@@ -819,6 +863,13 @@ static int test_observer_server(void)
 		goto done;
 	}
 
+	r = coap_resource_notify(&server_resources[1]);
```

Comment:
I am not sure what you are testing here, second resource is well-known/core.

Reply:
Coming Monday I will provide you complete analysis.

Reply:
I am able to solve this.